ephios/plugins/basesignup/templates/basesignup/participation_card_inline.html

```diff
@@ -1,17 +1,36 @@
+{% load guardian_tags %}
 {% load user_extras %}
 {% load i18n %}
 
+{% if request.user.is_authenticated %}
+    {% get_obj_perms request.user for event as "event_perms" %}
+{% endif %}
+
 <div id="popover-title-{{ participation.id }}" class="d-none">
     <span class="text-dark pe-1">{{ participation.participant.icon }}</span>
     {{ participation.participant }}
 </div>
 
 <div id="popover-content-{{ participation.id }}" class="d-none">
-    {% for qualification in participation.participant.qualifications|get_relevant_qualifications %}
-        <span class='badge bg-secondary text-dark'>{{ qualification }}</span>
-    {% empty %}
-        <em>{% translate "none" %}</em>
-    {% endfor %}
+    {% if participation.individual_start_time or participation.individual_end_time %}
+        <div class="mb-3">
+            <i class="fas fa-user-clock"></i>
+            <strong>{{ participation.get_start_end_time_display }}</strong>
+        </div>
+    {% endif %}
+    {% if participation.comment %}
+        <div class="mb-3">
+            <i class="fas fa-comment"></i>
+            <span>{{ participation.comment }}</span>
+        </div>
+    {% endif %}
```

Comment:
So this shows comment and times to everyone with read access to the event. Do we want that?

Reply:
time could be interesting/helpful, but I wouldn't show the comment for everyone